src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$main-accent: #060b35;
+$blue-accent: #00acdc;
+$secondary: #616070;
```

Comment:
secondary what? Also, let's rename variables, they should show their purpose, not their content.